test/unit/account/credit_card_test.rb

```diff
@@ -98,12 +98,12 @@ class Account::CreditCardTest < ActiveSupport::TestCase
   end
 
   test '#credit_card_exires_on_with_default' do
-    Timecop.freeze(Time.utc(2017,8,30))
+    travel_to(Time.utc(2017,8,30))
```

Comment:
it seems using a block makes more sense here

Reply:
Done